src/map/status.c

```diff
@@ -5008,7 +5008,10 @@ static int status_calc_watk(struct block_list *bl, struct status_change *sc, int
 		watk += sc->data[SC_FLASHCOMBO]->val2;
 	if (sc->data[SC_CATNIPPOWDER])
 		watk -= watk * sc->data[SC_CATNIPPOWDER]->val2 / 100;
-
+#ifdef RENEWAL
+	if (sc->data[SC_SHOUT])
+		watk += sc->data[SC_SHOUT]->val2;
```

Comment:
I have changed this watk to batk for a better use:

```
	if (sc->data[SC_SHOUT])
		batk += sc->data[SC_SHOUT]->val2;	
```
		
But this should be added under the batk section, therefore after (line 5065 in my case - may differ):
```
	if (sc->data[SC_SHRIMP])
		batk += batk * sc->data[SC_SHRIMP]->val2 / 100;	
```